pkg/watcher/reconciler/dynamic/dynamic.go

```diff
@@ -410,5 +465,78 @@ func (r *Reconciler) streamLogs(ctx context.Context, o results.Object, logType,
 		Err: writer,
 	}, logChan, errChanRepeater)
 
+	logger.Debugw("Exiting streaming logs",
```

Comment:
while @pmacik is testing for us, let's temporarily change this to an Info level log

that way, if need be, we can look for these message in the results watcher log gathered in the artifacts from his e2e tests